evm/test/Upgrades.t.sol

```diff
@@ -41,6 +41,7 @@ contract TestUpgrades is Test, IManagerEvents, IRateLimiterEvents {
         0xcfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0;
     WormholeSimulator guardian;
     uint256 initialBlockTimestamp;
+    uint8 constant FAST_CONSISTENCY_LEVEL = 200;
```

Comment:
Why are we using this consistency level in the tests? Intrigued more than anything

Reply:
this is code for instant finality. for testnet @evan-gray mentioned this would better.